### DISCUSSION TITLE
Soon ready to test toolchain

### DISCUSSION THREAD

**Discussion by micttyl** [carbon-language/carbon-lang#2151] [Toolchain]
Are we going to recycle Explorer's testdata for toolchain? If so, doesn't it matter that we cannot accumulate cases that are [not implemented](https://github.com/carbon-language/carbon-lang/wiki/Are-we-explorer-yet%3F) in the current Explorer?

**Comment by jonmeow** [comment]
We haven't implemented the toolchain yet, and may try to share, but right now the testdata is under /explorer because it's tool-specific. If we share test cases, we'd want to put them somewhere else (maybe /common?) and would need to think about sharing of cases.
